config/apply/ccsm/translate.js

```diff
@@ -245,6 +245,64 @@ const customHistologyCodes = {
   }
 };
 
+// ## ExcisionResults
+// >Can be null.
+
+// Maps to category values in ECT, 14006 - CCS - TRANSCRIBED EXCISION RESULTS
+
+// ECT.14006.1 - Normal
+// ECT.14006.3 - CIN1
+// ECT.14006.4 - CIN2
+// ECT.14006.5 - CIN3
+// ECT.14006.6 - AIS
+// ECT.14006.10 - Squamous Cell Carcinoma
+// ECT.14006.11 - Adenocarcinoma
+// ECT.14006.12 - Margins Negative
+// ECT.14006.13 - Margins Positive
+// ECT.14006.99 - Other
+const customExcisionCodes = {
```

Comment:
I decided to list each of the fields for completeness and in case they needed to be mapped later. But you're right, it may be cleaner to only include the three items in the object. This also could clean up some of the logic here: https://github.com/ccsm-cds-tools/ccsm-cds-service/blob/excision-results/config/apply/ccsm/translate.js#L414

Reply:
I took your suggestion and added https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/20/commits/69b12d7a4149ed2554a92a71d9775aa36c88e6a6. Does this align with what you were thinking for how to represent this object?

This also allowed me to more cleanly define the `ExcisionResultsShowAisOrCancer` boolean, since we no longer have to redefine an array containing each of the Excision Results ID codes.